recipes/hdf4/linux_build.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+mkdir -vp ${PREFIX}/bin;
+
+ARCH="$(uname 2>/dev/null)"
+
+export CFLAGS="-m64 -pipe -O2 -march=x86-64 -fPIC"
```

Comment:
It's OK to add to CFLAGS, but please don't set them this way.  As an example, this line breaks 32-bit builds (which conda-forge does not currently support, but which the Continuum internal build system does.  Continuum would not be able to use this recipe without modification.

If you do set CFLAGS or CXXFLAGS, please, always set them separately, and do something like:

export CFLAGS="${CFLAGS} -I/example/flag"
export CXXFLAGS="${CXXFLAGS} ${CFLAGS}"

This is so that you always avoid clobbering someone else's setting.  Note that CXXFLAGS needs to keep its prior setting, too, in addition to inheriting from CFLAGS.